### ISSUE TITLE
How It's works "Normalize" function

### ISSUE THREAD

**Issue by Jes46** [albumentations-team/albumentations#1730]
Hi, everyone. 
I am doing a university work and It is necessary to know how the **_"Normalize"_** function works much better than how It is explained in the docomentation.

Can anyone help me?

How exactly do "mean" parametrer and "std" parametrer work? They are involved in what type of normalization makes the function or if I specify what type of normalization don't matter the value of "mean" and "std" 

Thanks

**Comment by ternaus** [comment]
Does this code help?

https://github.com/albumentations-team/albumentations/blob/main/albumentations/augmentations/functional.py#L87-L107

> If I use mean(0.0, 0.0, 0.0) and std=(1.0, 1.0, 1.0)
> What type of normalization I'm doing?

For standard normalization:
```
img = ((img / max_value) - mean) / std
```

if `mean = (0, 0, 0)` and `std = (1, 1, 1)`

and image has `np.uint8` type, which has 255 as max value 
=>
```
img = img / 255
```

Which effectively rescales image from range `[0, 255]` to `[0, 1]`

I believe, YOLO uses this type of the normalization.

**Comment by Jes46** [comment]
Yes, that was the reason I asked for. I'm using YOLOv8 and I need to explain the normalization.
Thanks so much.
